src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/OracleEngine.java

```diff
@@ -1035,6 +1035,11 @@ protected void addFks(DbEntity entity) throws DatabaseEngineException {
 
     @Override
     protected boolean checkConnection(final Connection conn) {
+        return checkConnection(conn, this.properties.getCheckConnectionTimeout());
+    }
+
+    @Override
+    protected boolean pingConnection(final Connection conn) {
```

Comment:
I think this isn't needed for oracle since we are always entering in the checkconnection below that used the conn.isValid(timeout). Am I wrong?